packages/core/src/rules/common/assertions/__tests__/asserts.test.ts

```diff
@@ -624,5 +624,26 @@ describe('oas3 assertions', () => {
         ).toEqual({ isValid: true, location: baseLocation.key() });
       });
     });
+
+    describe('function', () => {
+      it('node must have at least one property from predefined list', () => {
+        const customFn = jest.fn((value: string[], options: any, location: Location) => {
+          expect(this).toEqual({});
```

Comment:
Please check if this test fails when removing `call`: https://github.com/Redocly/redocly-cli/pull/837/files#diff-1c79392a8637cdb873362603ed98b40fc02d36e8bf00241c0b6f2aee7a411d67R235